src/qibolab/platform.py

```diff
@@ -352,6 +352,123 @@ def create_tii_rfsoc4x2(runcard, address=None):
     return platform
 
 
+def create_tii_zcu111(runcard, address=None):
```

Comment:
As described for the runcards, even `create_` functions have been moved, exactly in the same locations (the only difference is that https://github.com/qiboteam/qibolab_platforms_qrc/ will be the permanent location in this case).